relayer/src/chain/cosmos/version.rs

```diff
@@ -63,19 +72,22 @@ impl TryFrom<VersionInfo> for Specs {
         // Get the Cosmos SDK version
         let sdk_version = parse_sdk_version(&raw_version)?;
         let ibc_go_version = parse_ibc_go_version(&raw_version)?;
+        let tendermint_version = parse_tendermint_version(&raw_version)?;
 
         trace!(
-            "parsed version specification for {} {}@{} -> SDK={}; Ibc-Go status={:?}",
+            "parsed version specification for {} {}@{} -> SDK={}; Ibc-Go status={:?}; Tendermint={:?}",
```

Comment:
The log line could use KV pairs more, rather than format them into the message.

Reply:
fixed, thank you for catching that!

[1570530](https://github.com/informalsystems/ibc-rs/pull/2302/commits/1570530d368dffb907da85104e9e303a904bed7d)